docs/user-guide/run-benchmark.md

```diff
@@ -47,12 +47,12 @@ genai-bench benchmark \
             --api-base "http://localhost:8180" \
             --api-model-name "/models/Phi-3-vision-128k-instruct" \
             --model-tokenizer "/models/Phi-3-vision-128k-instruct" \
-            --task image-to-text \
+            --task image-text-to-text \
```

Comment:
Can we add a note above the shell command that, user can choose not to pass in any `--traffic-scenario` for HF dataset, the benchmark will not do any operations to the image and will pass them directly to the server.

Reply:
Ignore this comment. For now we don't have a clean design on how to control this yet.